gem2rpm/default.spec.erb

```diff
@@ -6,6 +6,7 @@ LICENSES = JSON.load(File.read(File.join(root, 'licenses.json')))
 
 config.rules[:doc] << /\/?CHANGES/
 config.rules[:doc] << 'sample'
+config.rules[:doc] << 'example'
 config.rules[:doc] << /\/?SECURITY/
```

Comment:
So I found out why this didn't appear to work anymore: https://github.com/fedora-ruby/gem2rpm/issues/115